package.json

```diff
@@ -33,18 +36,18 @@
     "dependencies": {
         "@rollup/pluginutils": "^4.1.0",
         "commander": "^4.1.1",
-        "glob": "^7.1.6",
-        "minimatch": "^3.0.4",
-        "pofile": "^1.1.1",
+        "glob": "^7.2.3",
+        "minimatch": "^9.0.0",
+        "pofile": "^1.1.4",
         "svelte": "3.32.3"
     },
     "devDependencies": {
-        "@oat-sa/eslint-config-tao": "^0.1.0",
+        "@oat-sa/eslint-config-tao": "^2.0.0",
         "@oat-sa/prettier-config": "^0.1.1",
-        "eslint": "^6.8.0",
-        "eslint-plugin-jest": "^23.8.2",
-        "eslint-plugin-jsdoc": "^22.0.1",
-        "jest": "^25.1.0"
+        "eslint": "^8.39.0",
+        "eslint-plugin-jest": "^27.2.1",
+        "eslint-plugin-jsdoc": "^40.1.1",
```

Comment:
Wasn't it the point of https://github.com/oat-sa/eslint-config-tao/releases/tag/v2.0.0 that we no longer need to include `eslint-plugin-*` in all projects?